bundles/org.openhab.ui.habot/src/main/java/org/openhab/ui/habot/rest/internal/HABotResource.java

```diff
@@ -71,6 +77,10 @@
  * @author Yannick Schaus - Initial contribution
```

Comment:
I guess you forgot to add a new `@author` entry like you did in the other files.

Reply:
Added!